src/platforms/hosted/bmp_libusb.c

```diff
@@ -19,13 +19,72 @@
 
 /* Find all known usb connected debuggers */
 #include "general.h"
+#if defined(_WIN32) || defined(__CYGWIN__)
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#include <stdint.h>
+
+#include "ftd2xx.h"
+#else
 #include <libusb.h>
+#include "libftdi1/ftdi.h"
+#endif
 #include "cli.h"
 #include "ftdi_bmp.h"
 #include "version.h"
+#include "probe_info.h"
 
 #define NO_SERIAL_NUMBER "<no serial number>"
 
+void get_bmp_product_version_string(struct libusb_device_descriptor *device_descriptor, libusb_device *device,
```

Comment:
`struct libusb_device_descriptor` -> `libusb_device_descriptor_s` please (and throughout the file)

Reply:
All instances in file were changed.